manifests/plugin/pcie_errors.pp

```diff
@@ -0,0 +1,15 @@
+class collectd::plugin::pcie_errors (
+  Enum['present', 'absent'] $ensure                   = 'present',
+  Optional[String]          $source                   = 'sysfs',
+  Optional[String]          $access_dir               = undef,
+  Optional[Boolean]         $report_masked            = false,
+  Optional[Boolean]         $persistent_notifications = false,
```

Comment:
does Optional make sense here? In most cases undef isn't a useful value for a Boolean variable.